torch/csrc/lazy/core/ir_builder.h

```diff
@@ -269,8 +269,10 @@ static inline NodePtr MakeSizeDiv(const Value& a, const Value& b) {
 
 inline Value GetSymIntValue(c10::SymInt a) {
   return Value(
-      dynamic_cast<torch::lazy::SymIntNodeImpl*>(a.toSymIntNodeImpl().get())
-          ->node_,
+      a.is_symbolic() ? dynamic_cast<torch::lazy::SymIntNodeImpl*>(
```

Comment:
I'd suggest to replace this `expect_int` I think we all agreed (cc: @wconstab ) we aren't going to create real symints in LTC. XLA won't be using codegen either

Reply:
I guess XLA will be overriding this function? then I am ok with `expect_int`.  the way it is seems ok too, i guess it serves as a guide for how to handle symints in XLA but also confuses people since the rest of TS backend doesn't have DS functionality

Reply:
I'm going to try not to wobble this code for now, if we want to rip out more of the embryonic LTC dynamic shapes stuff, let's do it in a separate pr